src/main/java/org/opensearch/performanceanalyzer/hwnet/NetworkInterface.java

```diff
@@ -98,6 +101,8 @@ public void putAll(NetInterfaceMetrics m) {
 
     private static LinuxIPMetricsGenerator linuxIPMetricsGenerator = new LinuxIPMetricsGenerator();
 
+    private static final Pattern stringPattern = Pattern.compile("[ \\t]+");
```

Comment:
static final variables follow all caps. ```STRING_PATTERN```

Also, can we define the splitter here directly?
```
private static final Splitter STRING_PATTERN_SPLITTER = Splitter.on(Pattern.compile("[ \\t]+"));
.
.
.
List<String> toks = STRING_PATTERN_SPLITTER.splitToList(line);
```